Polyfills/Canvas/Source/Context.cpp

```diff
@@ -0,0 +1,694 @@
+            {"lightpink", 0xffb6c1},
+            {"lightsalmon", 0xffa07a},
+            {"lightseagreen", 0x20b2aa},
+            {"lightskyblue", 0x87cefa},
+            {"lightslategray", 0x778899},
+            {"lightslategrey", 0x778899},
+            {"lightsteelblue", 0xb0c4de},
+            {"lightyellow", 0xffffe0},
+            {"lime", 0x00ff00},
+            {"limegreen", 0x32cd32},
+            {"linen", 0xfaf0e6},
+            {"magenta", 0xff00ff},
+            {"maroon", 0x800000},
+            {"mediumaquamarine", 0x66cdaa},
+            {"mediumblue", 0x0000cd},
+            {"mediumorchid", 0xba55d3},
+            {"mediumpurple", 0x9370db},
+            {"mediumseagreen", 0x3cb371},
+            {"mediumslateblue", 0x7b68ee},
+            {"mediumspringgreen", 0x00fa9a},
+            {"mediumturquoise", 0x48d1cc},
+            {"mediumvioletred", 0xc71585},
+            {"midnightblue", 0x191970},
+            {"mintcream", 0xf5fffa},
+            {"mistyrose", 0xffe4e1},
+            {"moccasin", 0xffe4b5},
+            {"navajowhite", 0xffdead},
+            {"navy", 0x000080},
+            {"oldlace", 0xfdf5e6},
+            {"olive", 0x808000},
+            {"olivedrab", 0x6b8e23},
+            {"orange", 0xffa500},
+            {"orangered", 0xff4500},
+            {"orchid", 0xda70d6},
+            {"palegoldenrod", 0xeee8aa},
+            {"palegreen", 0x98fb98},
+            {"paleturquoise", 0xafeeee},
+            {"palevioletred", 0xdb7093},
+            {"papayawhip", 0xffefd5},
+            {"peachpuff", 0xffdab9},
+            {"peru", 0xcd853f},
+            {"pink", 0xffc0cb},
+            {"plum", 0xdda0dd},
+            {"powderblue", 0xb0e0e6},
+            {"purple", 0x800080},
+            {"red", 0xff0000},
+            {"rosybrown", 0xbc8f8f},
+            {"royalblue", 0x4169e1},
+            {"saddlebrown", 0x8b4513},
+            {"salmon", 0xfa8072},
+            {"sandybrown", 0xf4a460},
+            {"seagreen", 0x2e8b57},
+            {"seashell", 0xfff5ee},
+            {"sienna", 0xa0522d},
+            {"silver", 0xc0c0c0},
+            {"skyblue", 0x87ceeb},
+            {"slateblue", 0x6a5acd},
+            {"slategray", 0x708090},
+            {"slategrey", 0x708090},
+            {"snow", 0xfffafa},
+            {"springgreen", 0x00ff7f},
+            {"steelblue", 0x4682b4},
+            {"tan", 0xd2b48c},
+            {"teal", 0x008080},
+            {"thistle", 0xd8bfd8},
+            {"tomato", 0xff6347},
+            {"turquoise", 0x40e0d0},
+            {"violet", 0xee82ee},
+            {"wheat", 0xf5deb3},
+            {"white", 0xffffff},
+            {"whitesmoke", 0xf5f5f5},
+            {"yellow", 0xffff00},
+            {"yellowgreen", 0x9acd32}};
+        
+        if (str[0] == '#' && str.length() == 4)
+        {
+            unsigned int components[4];
+            int count = sscanf(str.c_str(), "#%1x%1x%1x", &components[0], &components[1], &components[2]);
```

Comment:
Try using sscanf_s (https://en.cppreference.com/w/c/io/fscanf)?

Reply:
Will do tests and push it in next PR if interesting results.